src/common/components/Typography/Typography.tsx

```diff
@@ -1,37 +1,49 @@
 import * as React from 'react'
+import cn from 'classnames'
 
-import { typography } from '../../theme/typography'
-import { colors } from '../../theme/colors'
-
-import * as Styles from './Typography.styles'
-
-type TypographyVariant = 'h1' | 'h2' | 'h3' | 'body1' | 'body2' | 'body3'
-type TypographyWeight = 400 | 500 | 800
+export enum TypographyVariant {
+  h1,
+  h2,
+  h3,
+  body1,
+  body2,
+  body3,
+}
 
-export interface TypographyProps {
+interface ITypographyProps {
   children: string
-  color?: string
   variant?: TypographyVariant
-  weight?: TypographyWeight
+  customFontWeight?: string
+  customColor?: string
   uppercase?: boolean
+  className?: string
 }
 
-const Typography: React.FC<TypographyProps> = ({
+const Typography: React.FC<ITypographyProps> = ({
```

Comment:
I think it's not a good idea to use React.FC. We need to keep the same style throught the project.

Reply:
Updated